	if __OPTIMIZE__ isnt defined, the comparison in this header fails

	Already fixed in upstream cvs

--- glibc-2.3.4/include/features.h
+++ glibc-2.3.4/include/features.h
@@ -262,7 +262,7 @@
 # define __USE_REENTRANT	1
 #endif
 
-#if _FORTIFY_SOURCE > 0 && __GNUC_PREREQ (4, 1) && __OPTIMIZE__ > 0
+#if _FORTIFY_SOURCE > 0 && __GNUC_PREREQ (4, 1) && defined(__OPTIMIZE__)
 # if _FORTIFY_SOURCE == 1
 #  define __USE_FORTIFY_LEVEL 1
 # elif _FORTIFY_SOURCE > 1
